gulpfile.js

```diff
@@ -29,39 +29,39 @@ const RELEASE_OSX_IMAGE_ICON = RELEASE_IMAGE_ICON + '.icns';
 const RELEASE_WIN_IMAGE_ICON = RELEASE_IMAGE_ICON + '.ico';
 
 const RELEASE_SETTINGS = {
-  dir: '.',
-  name: appConfig.name,
-  out: appConfig.releasePath,
-  version: '0.36.0',
-  'app-version': appConfig.version,
-  'version-string': {
-    ProductVersion: appConfig.version,
-    ProductName: appConfig.name,
-  },
-  ignore: '/node_modules/(' + RELEASE_IGNORE_PKGS.join('|') + ')',
-  appPath: 'build/browser/main.js',
-  overwrite: true,
-  force: true,
-  asar: true,
-  prune: true
+    dir: '.',
```

Comment:
could you remove these extras spaces that were introduced? hard to see what's going on in this PR